packages/web-components/src/components/ic-status-tag/readme.md

```diff
@@ -9,6 +9,7 @@
 
 | Property             | Attribute    | Description                                                     | Type                                              | Default     |
 | -------------------- | ------------ | --------------------------------------------------------------- | ------------------------------------------------- | ----------- |
+| `announced`          | `announced`  | If 'true', the component will act as an 'aria-live' region.     | `boolean`                                         | `false`     |
```

Comment:
Think you need to run the build again to update this readme file. `npm run build`